### ISSUE TITLE
Implicit override warnings reported for _type_checker_internals.pyi

### ISSUE THREAD

**Issue by correctmost** [microsoft/pyright#10484]
**Describe the bug**
`reportImplicitOverride` warnings are reported for an internal `_type_checker_internals.pyi` file when type checking.

**Code or Screenshots**

test.py:
```python
x=1
```

pyrightconfig.json:
```json
{
  "reportImplicitOverride": true
}
```

Running `npx pyright test.py` produces these warnings:

```
~/.npm/_npx/110e52990071af13/node_modules/pyright/dist/typeshed-fallback/stdlib/_typeshed/_type_checker_internals.pyi
  ~/.npm/_npx/110e52990071af13/node_modules/pyright/dist/typeshed-fallback/stdlib/_typeshed/_type_checker_internals.pyi:40:9 - error: Method "items" is not marked as override but is overriding a method in class "Mapping[str, object]" (reportImplicitOverride)
  ~/.npm/_npx/110e52990071af13/node_modules/pyright/dist/typeshed-fallback/stdlib/_typeshed/_type_checker_internals.pyi:41:9 - error: Method "keys" is not marked as override but is overriding a method in class "Mapping[str, object]" (reportImplicitOverride)
  ~/.npm/_npx/110e52990071af13/node_modules/pyright/dist/typeshed-fallback/stdlib/_typeshed/_type_checker_internals.pyi:42:9 - error: Method "values" is not marked as override but is overriding a method in class "Mapping[str, object]" (reportImplicitOverride)
```

**VS Code extension or command-line**
Command-line version 1.1.401

**Comment by jorenham** [comment]
For what it's worth; this does not occur when running pyright using https://github.com/RobertCraigie/pyright-python. But I did encounter these same errors with [`pyright-action`](https://github.com/RobertCraigie/pyright-python), for example in https://github.com/scipy/scipy-stubs/actions/runs/15169287965/job/42655058438?pr=521.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.402.
